src/main/java/jalau/at18/architects/GreenCard.java

```diff
@@ -4,8 +4,8 @@ public class GreenCard extends Card {
     private String color;
     private ScientificSymbol scientificSymbol;
 
-    public GreenCard(String color, ScientificSymbol scientificSymbol) {
-        this.color = color;
+    public GreenCard(ScientificSymbol scientificSymbol) {
+        this.color = "Green";
```

Comment:
If moved to super class, here we could just call `super("Green")` where `Card` defines a constructor to set color